src/bp-forums/classes/class-bp-forums-component.php

```diff
@@ -424,5 +426,134 @@ public function bb_group_forums_set_title_tag( $title ) {
 
 			return $title;
 		}
+
+		/**
+		 * Admin bar menu for forum and topic.
+		 *
+		 * @since BuddyBoss [BBVERSION]
+		 */
+		public function bb_forums_admin_bar_menu() {
+			global $wp_admin_bar;
+			if ( bp_is_single_item() && bp_is_group() && bp_current_action() === 'forum' && ! bp_is_group_forum_topic() ) {
```

Comment:
```suggestion
			if ( bp_is_single_item() && bp_is_group() && 'forum' === bp_current_action() && ! bp_is_group_forum_topic() ) {
```

Reply:
@krunal-buddyboss What happens if someone has a different singular slug instead of a `forum`?

Please test it out with a different name.

Condition should be like `get_option( '_bbp_forum_slug', 'forum' ) === === bp_current_action()`

Reply:
@chetansatasiya  I have fixed it.